Gemfile

```diff
@@ -4,6 +4,7 @@ ruby '2.2.3' # this should match `.ruby-version` and docs/setup.md
 gem 'active_model_serializers'
 gem 'activeadmin', github: 'activeadmin'
 gem 'acts_as_list'
+gem 'acts-as-taggable-on', '~> 3.4'
```

Comment:
afaik, the `~>` syntax is significant in terms of API/ABI compatability. In this case, at least version 3.4 but nothing 4.0 or higher.